### PR TITLE
Fix `AESND_Reset` hanging (#140)

### PR DESCRIPTION
**Pull request by muff1n1634**

### CODE REVIEW

File: libaesnd/dspcode/dspmixer.s [hunk 1]
```diff
@@ -157,7 +157,8 @@ recv_cmd:
 	cmpi	$acc1.m,#0x0100
 	jeq		send_samples
 	
-	cmpi	$acc1.m,#0xdead
```
Comment (DacoTaco) [inline libaesnd/dspcode/dspmixer.s:160]: could you add a comment explaining why this is different from the others? don't make it long, just explain its sign extending

**Comment by DacoTaco** [comment]
odd as it is, i got the following dol once i added gamecube controller support to the test code. it still froze on exit. unsure why. 
[aestest.zip](https://github.com/devkitPro/libogc/files/10891561/aestest.zip)
did it work for you?

**Comment by muff1n1634** [comment]
the provided dol did freeze on exit, but recompiling the code with the fixed version of aesndlib didn't freeze; perhaps you didn't recompile aesndlib?

**Comment by DacoTaco** [comment]
ill recheck tonight, but i did a make clean and make install twice before doing a make clean & make of the test case code

**Comment by DacoTaco** [comment]
found & merged fix in #143 
thanks for your help!

**Pull request closed without merging**